apps/mail/components/home/footer.tsx

```
@@ -1,246 +1,214 @@
+      { name: 'AI Chat', href: '/ai-chat' },
+      { name: 'Bulk Actions', href: '/bulk-actions' },
+    ]
+  }
+};
+
 const socialLinks = [
+  {
+    name: 'GitHub',
+    href: 'https://github.com/Mail-0/Zero',
+    icon: (props: any) => (
+      <svg viewBox="0 0 24 24" {...props}>
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)